packages/common/package.json

```diff
@@ -1,44 +1,44 @@
 {
-  "name": "@prosopo/common",
-  "version": "0.1.17",
-  "description": "Prosopo common library",
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
-  "scripts": {
-    "clean": "tsc --build --clean",
-    "build": "tsc --build --verbose",
-    "test": "mocha -r ts-node/register 'tests/**/*.ts'",
-    "lint": "npx eslint .",
-    "lint:fix": "npx eslint . --fix"
-  },
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "@polkadot/keyring": "12.3.2",
-    "@polkadot/util": "12.3.2",
-    "@polkadot/util-crypto": "12.3.2",
-    "@types/react": "^18.0.25",
-    "consola": "^2.15.3",
```

Comment:
Isn't `consola` still required?

Reply:
erm yeah, not sure why that got removed. My bad!